src/main/kotlin/minesweeper/domain/BlockCreator.kt

```diff
@@ -0,0 +1,20 @@
+package minesweeper.domain
+
+class BlockCreator(
+    height: Int,
+    width: Int,
+    mineCount: Int,
+    val blocks: List<Block> = (createNormalBlocks(height, width, mineCount) + createMineBlocks(mineCount)).shuffled()
+) {
```

Comment:
```suggestion
class BlockCreator(
    height: Int,
    width: Int,
    mineCount: Int,
) {
    val blocks: List<Block> = (createNormalBlocks(height, width, mineCount) + createMineBlocks(mineCount)).shuffled()
```
생성자로 받지 않는게 좀 더 자연스러워 보여요.
이렇게 하면 `createXXBlocks`가 static으로 선언되지 않아두 되겠네요 :)